app/celer-g4/ActionInitialization.cc

```diff
@@ -73,7 +82,7 @@ void ActionInitialization::Build() const
     CELER_LOG_LOCAL(status) << "Constructing user action";
 
     // Primary generator emits source particles
-    if (auto hepmc_gen = GlobalSetup::Instance()->hepmc_gen())
+    if (auto const& hepmc_gen = GlobalSetup::Instance()->hepmc_gen())
```

Comment:
That seems that it would be a superfluous change (i.e. redundant with the change in the signature).

Reply:
The '&' is necessary, the `const` is redundant.